### ISSUE TITLE
Are scenes supported ?

### ISSUE THREAD

**Issue by sander76** [studioimaginaire/phue#50]
Scenes are pretty well documented in the API. Am I missing something or are scenes not supported ?

**Comment by natcl** [comment]
They are not supported yet.  I'll be working on Sensor support next, Scenes could be next.  If you want it faster, you're welcome to submit a pull request.

**Comment by sander76** [comment]
Not sure whether I am able to, but I'll give it a shot !

**Comment by avzdk** [comment]
I too would like to be able to activate preset scenes.

**Comment by ledge74** [comment]
Scenes support would be great ! Sadly I don't have the skills to do it myself :(

**Comment by natcl** [comment]
Closed by #73

**Comment by sdague** [comment]
Could we get a 0.9 release with this in it so I can start consuming it from Home Assistant components?

**Comment by natcl** [comment]
Yup will release soon, need to remember how to submit the package ;)

**Comment by sdague** [comment]
python setup.py sdist upload

Also, if you need help with it, let me know, I'm happy to help. I would need github perms to set a tag (optional, but it seems good to keep tags in sync) and pypi perms to do package upload.

Thanks again.
